tests/components/devolo_home_network/__init__.py

```diff
@@ -28,5 +28,6 @@ def configure_integration(hass: HomeAssistant) -> MockConfigEntry:
 
 async def async_connect(self, session_instance: Any = None):
     """Give a mocked device the needed properties."""
+    self.mac = DISCOVERY_INFO["properties"]["PlcMacAddress"]
```

Comment:
Use dataclass properties now
```suggestion
    self.mac = DISCOVERY_INFO.properties["PlcMacAddress"]
```